readme.md

```diff
@@ -78,10 +78,13 @@ Markdown template for each commit entry in release notes. Available replacements
 ### exclude
 
 Default: `''` <br>
-Example: `'^Meta:'`
+Example: `'^Meta:'` <br>
+Example: `true`
 
 Regex to exclude commits based on their title (don't include the initial and final `/`).
 
+Setting this to `true` will enable the default preset, which may change over time, but is currently: `/^meta|^document|^lint|^refactor|readme|dependencies|^v?\d+\.\d+\.\d+/i`
```

Comment:
Wondering if this is going to bite the uses, as the input takes the string form of a regex (without the leading and trainling `/`). Also as they can not pass in flags for the regex as well.

Reply:
I was thinking the same, but it says ``don't include the initial and final `/`)`` just in the previous sentence.

Reply:
That should be cool then (except for the flag part)

Reply:
Yeah the best solution would be to support a full regex instead of just its textual string. Perhaps there’s an ~~app~~ module for that.